README.md

```diff
@@ -172,15 +172,15 @@ sudo apt install libicu-dev libtinfo-dev libgmp-dev
 sudo dnf install libicu-devel ncurses-devel
 ```
 
-#### Windows-specific pre-requirements (optional)
+#### Windows-specific pre-requirements
 
-In order to avoid problems with long paths on Windows you can do the following:
+In order to avoid problems with long paths on Windows you can do either one of the following:
 
-1. In the `Local Group Policy Editor`: `Local Computer Policy -> Computer Configuration -> Administrative Templates -> System -> Filesystem` set `Enable Win32 long paths` to `Enabled` (Works
-   only for Windows 10).
+1. Clone the `haskell-ide-engine` to the root of your logical drive (e.g. to
+   `C:\hie`). If this doesn't work try the second option.
 
-2. Clone the `haskell-ide-engine` to the root of your logical drive (e.g. to
-   `C:\hie`)
+2. In the `Local Group Policy Editor`: `Local Computer Policy -> Computer Configuration -> Administrative Templates -> System -> Filesystem` set `Enable Win32 long paths` to `Enabled` (Works
+   only for Windows 10). Also you need to configure git to allow longer paths by using unicode paths. To set this for all your git repositories use `git config --system core.longpaths true` (you probably need an administrative shell for this) or for just this one repository use `git config core.longpaths true`.
```

Comment:
Re git the issue is less likely to being triggered cause `dist-newstyle` and `.stack-work` is ignored. So the limit could be even greater than the established by cabal. As you always will hit the compilation problem with a shorter path than the maximum for git i think we could remove the git section.